app/views/grade_entry_forms/_react_grades_table.js.jsx.erb

```diff
@@ -154,11 +154,7 @@
           this.setState({
             error: error
           });
-        }.bind(this),
-        // Close working indicator.
-        complete: function () {
-          document.getElementById('working').style.display = 'none';
-        }
```

Comment:
@Sm1th We shouldn't assume that the second one will complete after the first one, so I think we will need this one.